Plugins/TopSkyMaps.txt

```diff
@@ -2703,25 +2652,49 @@ FOLDER:R/D-AREAS
 COLOR:RESTR
 STYLE:SOLID
 ZOOM:9
+ACTIVE:ID:NO,NS,2R,3R,4R,5R,6R,8R,DD,GER,FF,GWR,A,B:*:*:*
```

Comment:
Tror ikke vi trenger å vise disse på CID. Bedre å legge de inn selv.

Reply:
Ullersmo Fengsel ligger inne i CTR på 1500 fot, og burde absolutt vises, særlig for tårnet. Da zoom value er såpass høy vil den ikke vises for ACC om de ikke zoomer langt inn. Den kan ev. fjernes for disse, men bør beholdes aktiv for APP og tårn CID etter min mening.

![image](https://github.com/Vatsim-Scandinavia/ENOR-Sectorfile/assets/89723211/8d9ad41c-e92b-4f4f-a1d5-682077657de9)

Reply:
Denne kan ligge inne på TWR og APP.